README.md

```diff
@@ -28,29 +28,62 @@ application, each tutorial is constructed as hybrid theory and programming in an
 interactive environment, removing the gap between theory and implementation.
 
 If you have comments, questions, or would like to contribute to the project
-please feel free to email [me](mailto:dgasmith@gatech.edu).
+please see our [contributor guidelines]().
 
 ### Getting Started
 
-New users can either follow the
-[Getting Started](https://github.com/psi4/psi4numpy/blob/master/Tutorials/01_Psi4NumPy-Basics/1a_Getting-Started.ipynb)
-notebook or follow the less verbose directions below.
+1. Obtain required software:
+- [Psi4](http://psicode.org/psi4manual/1.1/build_obtaining.html)
+- [Python](https://python.org) 2.7+
+- [NumPy](http://www.numpy.org) 1.7.2+
+- [Scipy](https://scipy.org) 0.13.0+
+ 
+2. Update Psi4 to the development version with Conda:
+```
+conda update psi4 -c psi4/label/dev
+```
 
-To begin using Psi4NumPy:
-- Obtain recent versions of Psi4, NumPy, and SciPy (see Requirements for version details)
-- Locate the `Psi4_install/lib` directory and execute the following: ```export PYTHONPATH=Psi4_install/lib```
+3. Link PsiAPI
+    1. Find appropriate paths
```

Comment:
This might be confusing, since if you install with conda you don't actually need to do this as psi4 is already in the python path, once the appropriate environment has been activated.

The steps you have detailed will enable will allow psi4 to be used, but I think there may be some runtime issues if the environment is not activated. For example, if numpy is not installed in the root environment. 

It might be a better idea to detail how to enable psi4 if compiled from source, or if obtained from conda separately to avoid confusion. Such as:

```
3. Enable psi4 to be used
    - If psi4 was installed using conda:
       `source activate psi4env`
     - If psi4 was compiled from source 
       `install_prefix/psi4/bin/psi4 --psiapi-path`
        ... and so on
```

I think we saw at the developer meeting that the process of getting psi4 running on your system is not always simple.